aiida/orm/utils/mixins.py

```diff
@@ -105,13 +107,45 @@ def _set_function_starting_line_number(self, function_starting_line_number):
         """
         self.set_attribute(self.FUNCTION_STARTING_LINE_KEY, function_starting_line_number)
 
+    @property
+    def function_number_of_lines(self):
+        """Return the number of lines of the wrapped function in its source file.
+
+        :returns: the number of lines or None
+        """
+        line_count = self.get_attribute(self.FUNCTION_NUMBER_OF_LINES_KEY)
+
+        return line_count
+
+    def _set_function_number_of_lines(self, function_number_of_lines):
+        """Set the number of lines of the wrapped function in its source file.
+
+        :param function_number_of_lines: the number of lines
+        """
```

Comment:
Maybe add a check that it is a positive integer?

Reply:
I don't think it is needed, since this value is a `len`.